src/dataflow-types/types.rs

```diff
@@ -414,7 +437,23 @@ impl DataEncoding {
                 Some("text".to_owned()),
             )]),
         };
-        Ok(desc)
+        full_desc.add_cols(
+            desc.iter()
+                .map(|(name, typ)| (name.to_owned(), typ.to_owned())),
+        );
+        Ok(full_desc)
+    }
+
+    pub fn op_name(&self) -> &str {
+        match self {
+            DataEncoding::Bytes => "RawBytes",
+            DataEncoding::AvroOcf { .. } => "AvroValues",
```

Comment:
"AvroOcf"

Reply:
Removing `op_name` from this PR and applying your comments to the corresponding `op_name` in #2493 because I realized `op_name` is unnecessarily duplicated in the two PRs.